index.bs

```diff
@@ -2723,6 +2770,218 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 1. Return [=success=] with data null.
 
+</div>
+#### The browsingContext.locateNodes Command ####  {#command-browsingContext-locateNodes}
+
+The <dfn export for=commands>browsingContext.locateNodes</dfn> command returns a
+list of all nodes matching the specified locators.
```

Comment:
You are using the plural for locator here. I assume that this is a mistake, or shall we consider having the locator to be a list? Probably not given that it would make it hard to distinguish the results without having to put those into their own buckets as well.

Reply:
This is a good catch. In the original implementation, there was a notion of chaining locators and the `locator` property, was, in fact, a list, to allow round-trip minimization. I've been talked out of that proposal, in favor of a [more general solution for batching commands](https://github.com/w3c/webdriver-bidi/issues/447). I'll update the PR to reflect this.